crates/rust-analyzer/src/config.rs

```diff
@@ -844,6 +844,9 @@ impl Config {
             config.source_root_parent_map = source_root_map;
         }
 
+        // IMPORTANT : This holds as long as ` completion_snippets_custom` is declared `client`.
+        config.snippets = vec![];
```

Comment:
```suggestion
        config.snippets.clear();
```
No need to allocate a fresh vec